### PR TITLE
Update bots.yml

### PR DESCRIPTION
**Pull request by razseo**
Improve detect seznan images mail proxy.

### CODE REVIEW

File: regexes/bots.yml [hunk 1]
```diff
@@ -500,7 +500,15 @@
   producer:
     name: 'Google Inc.'
     url: 'http://www.google.com'
-
+    
+- regex: 'Seznam Email Proxy'
+  name: 'SeznamEmailProxy'
```
Comment (sgiehl) [inline regexes/bots.yml]: Guess you switched the content of `regex` and `name`

**Review by sgiehl — CHANGES_REQUESTED** [review]
Could you maybe also add a test for this bot, to prove if works?

File: regexes/bots.yml (hunk 1 above)
Comment (razseo) [reply regexes/bots.yml]: Mozilla/5.0 SeznamEmailProxy/2.0.174
--


Here is full user agent from my access log.

Comment (razseo) [reply regexes/bots.yml]: About test - you mean unit test? I never write it :(

**Comment by razseo** [comment]
So nobody need this?

File: regexes/bots.yml (hunk 1 above)
Comment (sgiehl) [reply regexes/bots.yml]: ok. will add it after merging

**Pull request merged**